presto-main/src/main/java/io/prestosql/server/ClusterStatsResource.java

```diff
@@ -57,9 +58,10 @@ public ClusterStats getClusterStats()
         long blockedQueries = 0;
         long queuedQueries = 0;
 
-        long activeNodes = nodeManager.getNodes(NodeState.ACTIVE).size();
```

Comment:
what's the rationale?

Reply:
I need to restart the coordinator without the user's awareness, so I will start the second coordinator in the same cluster to switch traffic, and the second coordinator will be closed after the restart. When the second coordinator started, I found that the number of active workers was incorrect, so I submit this commit for this reason. Thanks!

Reply:
Such scenario: "two coordinators registered to same discovery server" is not supported by Presto ATM, so workers and coordinator behavior is undefined. Definitely currently running queries will fail when coordinator is killed.

Correct way to do coordinator turnover is to have some kind of load balancer or name resolution for coordinator that would be routing to the currently active coordinator.

Reply:
![image](https://user-images.githubusercontent.com/19525211/57747153-d515de00-7706-11e9-8293-471725987d4d.png)
The picture shows the details of our restart ,i need restart the coordinator without user's awareness,that is ,the  running queries can not be affected. So i figure  out the picture  solution .Maybe there is better solution that i don't know,  i really appreciate it if you can tell me.Thanks !